### ISSUE TITLE
table widget selection icon

### ISSUE THREAD

**Issue by RyszardB** [ckeditor/ckeditor5-ui#435]
<img width="566" alt="screen shot 2018-07-20 at 13 57 40" src="https://user-images.githubusercontent.com/17498197/43001256-e7b4aa82-8c24-11e8-98c4-34177cc5a126.png">

This icon suggests that I can resize the table or move it.. I can not do anything like that. I can now only select the table and copy it.  Unfortunately, I have now no idea what the new icon should look like. :-(

**Comment by Reinmar** [comment]
cc @oleq @dkonopka

**Comment by oleq** [comment]
I agree this one isn't perfect but OTOH it will be replaced when we introduce the proper drag&drop of widgets anyway so I wouldn't worry much about that. I hope it happens sooner than later :p
